src/app.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const firstFilePath = path.join(__dirname, '/file.txt');
+const secondFilePath = path.join(__dirname, '/file-copy.txt');
+
+const copyFiles = (file, fileToCopy) => {
+  if (file !== fileToCopy) {
+    const firstFileData = fs.readFileSync(file, 'utf-8');
```

Comment:
You're using 'readFileSync' which is a blocking operation. This could lead to performance issues if the file is large. Consider using the async version 'fs.readFile' instead, which is non-blocking.